rkt/gc.go

```diff
@@ -238,14 +193,23 @@ func deletePod(p *pod) {
 		}
 		defer s.Close()
 
-		if err := cleanExitedGarbage(p, s); err != nil {
-			stderr("Received error performing GC for pod %q: %v", p.uuid, err)
-			stderr("Attempting forced GC of pod %q...", p.uuid)
-			if err := stage0.ForceMountGC(p.path(), p.uuid.String()); err != nil {
-				stderr("Forced GC of pod %q failed: %v\n", p.uuid, err)
-				return
+		// execute stage1's GC
+		stage1TreeStoreID, err := p.getStage1TreeStoreID()
+		if err != nil {
+			stderr("Error getting stage1 treeStoreID: %v", err)
+			stderr("Skipping stage1 GC")
+		} else {
+			stage1RootFS := s.GetTreeStoreRootFS(stage1TreeStoreID)
+			if err := stage0.GC(p.path(), p.uuid, stage1RootFS, globalFlags.Debug); err != nil {
+				stderr("Problem performing stage1 GC on %q: %v", p.uuid, err)
 			}
-			stderr("Forced GC of pod %q succeeded.", p.uuid)
+		}
+
+		// unmount all leftover mounts
+		stdout("GC of leftover mounts for pod %q...", p.uuid)
```

Comment:
Do we really need to output this everytime? Just an implementation detail, IMHO.